src/base/types/Gene.jl

```diff
@@ -4,15 +4,17 @@ Gene struct.
 # Fields
 ````
 id :: String
+name :: Maybe{String}
 notes :: Dict{String, Vector{String}}
 annotation :: Dict{String, Union{Vector{String}, String}}
 ````
 """
 mutable struct Gene
     id::String
+    name::Maybe{String}
     notes::Notes
     annotations::Annotations
 
-    Gene(id::String = ""; notes = Notes(), annotations = Annotations()) =
-        new(id, notes, annotations)
+    Gene(id::String = ""; name = nothing, notes = Notes(), annotations = Annotations()) =
+        new(id, name, notes, annotations)
```

Comment:
might be interesting to incorporate what we started using in SBML, basically there's a macro `@kwdef` that does this correctly:
https://github.com/LCSB-BioCore/SBML.jl/blob/master/src/structs.jl#L109

Not necessary now, just dropping the idea here. :D